docs/tutorials/java/audiobook/index.md

```diff
@@ -0,0 +1,982 @@
+   Using Git, you can initialize your repository:
+
+   ```sh
+   cd TTSWorker
+   git init
+   ```
+
+   Version control helps you manage changes and collaborate with others efficiently.
+   Just like Temporal, it provides check-in points so you can back up in time if needed to retry your project development without duplicating all your earlier work.
+
+3. **Create your build file**
+   In your root folder, create a build.gradle file and add the following contents.
+   Feel free to swap in Maven if you prefer:
+
+<!--SNIPSTART audiobook-project-java-Gradle-build-file-->
+[build.gradle](https://github.com/temporalio/build-audiobook-java/blob/main/build.gradle)
+```gradle
+group 'ttspackage'; version '0.0.1'
+repositories { mavenCentral() }
+apply plugin: 'java'
+sourceSets.main.java.srcDirs 'src'
+
+dependencies {
+    implementation 'io.temporal:temporal-sdk:1.22.2'
+    implementation 'org.slf4j:slf4j-nop:2.0.6' // logging suppression
+    implementation 'commons-io:commons-io:2.11.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.9.3'
+    implementation 'org.json:json:20210307'
+}
+
+// Run the App
+task run(type: JavaExec) {
+    classpath = sourceSets.main.runtimeClasspath
+    mainClass = 'ttspackage.TTSWorkerApp'
+    standardOutput = System.out
+}
+```
+<!--SNIPEND-->
+
+4. **Review your dependencies and run task**:
+   Your dependencies include Temporal's [Java SDK](https://github.com/temporalio/sdk-java), and a few basic libraries:
+
+   * com.squareup.okhttp3:okhttp:4.9.3: OkHttp is a basic HTTP client for network requests.
+   * org.json:json:20210307: Parse and manipulate JSON data.
+   * commons-io:commons-io:2.11.0: Perform file tasks with common input/output routines.
+   * org.slf4j:slf4j-nop:2.0.6: Minimizes unnecessary output with logging suppression.
+   * io.temporal:temporal-sdk:1.22.2: Add error mitigation.
+
+   The run task starts your TTS application.
+
+5. **Optionally, create a `bearer.sh` utility**:
+   Make it executable with `chmod +x`.
+   Store this wherever you keep similar secure items:
+
+   ```sh
+   #! /bin/sh
+
+   setenv OPEN_AI_BEARER_TOKEN 'your-secret-bearer-token'
+   ```
+
+   This 'bearer.sh' script sets your OpenAI bearer token as an environment variable.
+   You must `source /path/to/bearer.sh` to set the variable into your current shell.
+   Environment variables let you skip hard coding your bearer token into projects.
+
+:::note Your Bearer Token
+
+When using your application, you must set your `OPEN_AI_BEARER_TOKEN` environment variable in the same shell before execution.
+The application checks for the token and if it's not set, it will error.
+
+:::
+
+## Create your OpenAI conversion code
+
+Create an interface-implementation pair of files named `TTSActivities.java` and `TTSActivitiesImpl.java`.
+Add these to 'src/main/java/ttsworker' and include the following file contents.
+This class is responsible for the text-to-speech work in your project:
+
+<details>
+
```

Comment:
The Hover sentence could be made into a note or something similar. "Hover your cursor over the code block to reveal the copy-code option."

Reply:
You don't see the hover sentence unless you do the toggle, so it's part of that infrastructure. If you download and build, it will be more obvious.

Thanks for the check in.